platform/engineering/official_github_teams_process

```diff
@@ -0,0 +1,38 @@
+    * Select the team name after `vfs-*` or `vsp-*`.  An example would be `vagov-vfs-[team name here]`. Choose a name that will not change and is approved by DEPO. Team names should be connected to the team’s work and not the contractor working on the contract. 
+    * Each GitHub team needs to have DEPO one Maintainer; these are Admins on a team who can add and remove users. The Maintainer should be the team’s DEPO lead. 
+    * Maintainers should add all members to the team including both engineers and nonengineers. 
+* Inform the VSP Platform Support and the VSP Operations team of any new GitHub team by posting in the #vfs-platform-support  channel. 
+
+### Maintaining GitHub Teams 
+
+* Any new team members will need to be added to the `vagov-all-teams` GitHub team by a member of the Platform team as part of onboarding. This team will grant them base access to tools. New team members will not be allowed to join their *Official GitHub Team* until after their SAC adjudication returns a “favorable” response. For more information about the approval process, please see the [request for tools documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/orientation/request-access-to-tools.md#additional-orientation-steps-for-developers). 
+* Teams are responsible for adding and removing members from an *Official GitHub Team*. 
+    * Once approved, new team members should be added by the *Official GitHub Team’s* maintainers. 
+    * If a team member leaves an *Official GitHub Team* they need to be removed within three days of departing the team by one of the team’s maintainers and then: 
+        * Use the GitHub issue template “Offboarding Request” in the [va.gov-team repo](https://github.com/department-of-veterans-affairs/va.gov-team) to remove any access from the user.
+
+### Making Changes to an Official GitHub Team
+
+* No changes to an *Official GitHub Team* name can be made unless approved by DEPO, the VSP Platform Support, and the VSP Operations team.
+    * To request an *Official GitHub Team* name change, please post in the  #vfs-platform-support channel and provide:
+        * The existing and new team names
+        * An explanation for the name change
+        * A DEPO point of contact.  
+* A replacement maintainer needs to be appointed before any other maintainer on a team leaves their team to ensure there are at least two maintainers per team at any time.
```

Comment:
This may not be something that the maintainer controls. If someone leaves and they're kicked out of the VA org, the maintainer is just gone. We'll need to know how to deal with that internally if/when it happens.

Reply:
Updated this language a bit now that we are moving to DEPO only being admins. To your point, I agree. It will be hard to do that without being Org Maintainers.